js_modules/dagster-ui/packages/ui-core/src/runs/RunConfigDialog.tsx

```diff
@@ -39,99 +36,8 @@ import {doneStatuses} from './RunStatuses';
 import {RunTags} from './RunTags';
 import {RunsQueryRefetchContext} from './RunUtils';
 import {TerminationDialog} from './TerminationDialog';
-import {TimeElapsed} from './TimeElapsed';
-import {RunDetailsFragment} from './types/RunDetails.types';
 import {RunFragment} from './types/RunFragments.types';
 
-export const timingStringForStatus = (status?: RunStatus) => {
```

Comment:
This is sorta unrelated but I was reading this and cleaned it up a bit. 

- I renamed RunDetails to RunTimingDetails

- The RunConfigDialog was in this file even though it was totally unrelated to the RunDetails. I moved it to a separate file.